Investigate Community Test Suite Issues

# Failing Test Runs

[As of February 11th](https://typescript.visualstudio.com/TypeScript/_build/results?buildId=95729&view=results), it seems like community test runs have been failing in strange ways

> Test 3
> ```
> ##[error]Unhandled exception. System.IO.IOException: No space left on device
> System.IO.IOException: No space left on device
>    at System.IO.FileStream.WriteNative(ReadOnlySpan`1 source)
>    at System.IO.FileStream.FlushWriteBuffer()
>    at Microsoft.VisualStudio.Services.Agent.HostContext.Dispose(Boolean disposing)
>    at Microsoft.VisualStudio.Services.Agent.HostContext.Dispose()
>    at Microsoft.VisualStudio.Services.Agent.Worker.Program.Main(String[] args)
>    at System.IO.FileStream.WriteNative(ReadOnlySpan`1 source)
>    at System.IO.FileStream.FlushWriteBuffer()
>    at System.IO.FileStream.Flush(Boolean flushToDisk)
>    at System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder)
>    at System.Diagnostics.TextWriterTraceListener.Flush()
>    at Microsoft.VisualStudio.Services.Agent.HostTraceListener.WriteHeader(String source, TraceEventType eventType, Int32 id)
>    at Microsoft.VisualStudio.Services.Agent.HostTraceListener.TraceEvent(TraceEventCache eventCache, String source, TraceEventType eventType, Int32 id, String message)
>    at System.Diagnostics.TraceSource.TraceEvent(TraceEventType eventType, Int32 id, String message)
>    at Microsoft.VisualStudio.Services.Agent.Worker.Worker.RunAsync(String pipeIn, String pipeOut)
>    at Microsoft.VisualStudio.Services.Agent.Worker.Program.MainAsync(IHostContext context, String[] args)
> System.IO.IOException: No space left on device
>    at System.IO.FileStream.WriteNative(ReadOnlySpan`1 source)
>    at System.IO.FileStream.FlushWriteBuffer()
>    at System.IO.FileStream.Flush(Boolean flushToDisk)
>    at System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder)
>    at System.Diagnostics.TextWriterTraceListener.Flush()
>    at Microsoft.VisualStudio.Services.Agent.HostTraceListener.WriteHeader(String source, TraceEventType eventType, Int32 id)
>    at Microsoft.VisualStudio.Services.Agent.HostTraceListener.TraceEvent(TraceEventCache eventCache, String source, TraceEventType eventType, Int32 id, String message)
>    at System.Diagnostics.TraceSource.TraceEvent(TraceEventType eventType, Int32 id, String message)
>    at Microsoft.VisualStudio.Services.Agent.Tracing.Error(Exception exception)
>    at Microsoft.VisualStudio.Services.Agent.Worker.Program.MainAsync(IHostContext context, String[] args)
> ```
> Test 4
> `,##[error]We stopped hearing from agent Azure Pipelines 18. Verify the agent machine is running and has a healthy network connection. Anything that terminates an agent process, starves it for CPU, or blocks its network access can cause this error. For more information, see: https://go.microsoft.com/fwlink/?linkid=846610`

# Fluent UI?

It seems like in recent pull requests (last 30ish days?), we're not seeing Office UI Fabric getting mentioned in our community test suite runs: https://github.com/typescript-bot/TypeScript/pull/

Is the community test suite still testing Fluent UI (formerly Office UI Fabric)? There might be issues because the repo was moved to https://github.com/microsoft/fluentui, but it might be due to the above issue.

We should make sure we haven't lost coverage.

--- a/tests/cases/docker/office-ui-fabric/Dockerfile
+++ b/tests/cases/docker/office-ui-fabric/Dockerfile
@@ -12,9 +12,7 @@ COPY --from=typescript/typescript /typescript/typescript-*.tgz typescript.tgz
 WORKDIR /office-ui-fabric-react
 # Sync up all TS versions used internally to the new one (we use `npm` because `yarn` chokes on tarballs installed
 # into multiple places in a workspace in a short timeframe (some kind of data race))
-RUN npx lerna exec --stream --concurrency 1 -- npm install /typescript.tgz --exact --ignore-scripts
+RUN sed -i -e 's/"resolutions": {/"resolutions": { "\*\*\/typescript": "file:\/typescript\.tgz",/g' package.json
 RUN npx yarn
-# Perform scss task to generate scss code if present
-RUN npx lerna exec --stream --concurrency 1 --bail=false -- yarn run just scss
 ENTRYPOINT [ "npx" ]
 CMD [ "lerna", "exec", "--stream", "--concurrency", "1", "--loglevel", "error", "--bail=false", "--", "yarn", "run", "just", "ts"]

